ingest/scripts/standardize-lineage

```diff
@@ -5,11 +5,15 @@ Standardize lineage based on GISAID's subtype and lineage.
 import argparse
 import re
 import csv
-from sys import stdin
+from pathlib import Path
+import sys
 from typing import Iterable, Dict
 from augur.io.json import dump_ndjson, load_ndjson
-from augur.io.print import print_err
 
+SCRIPT_NAME = Path(sys.argv[0]).stem
+
+def print_err(*args):
+    print(f"[{SCRIPT_NAME}] ", *args, file=sys.stderr)
```

Comment:
I like this pattern for the curate scripts. I wonder if we should formalize this in Augur as well with the standard logging library.